### PR TITLE
Update token lists

### PR DESCRIPTION
**Pull request by TejasvOnly**

### CODE REVIEW

File: packages/core-services/src/db/migrations/items/006.trc20Token/index.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import {
+  migrateTokenDetailsChangeInDb,
+  migrateTokenIdChangeInDb,
+  resetContractTransactionBlockHeight,
+} from '../../helpers';
+import { IMigrationItem } from '../../types';
+
+const migration: IMigrationItem = {
+  id: '5',
+  name: 'ERC20 token list update migration',
```
Comment (muzaffarbhat07) [inline packages/core-services/src/db/migrations/items/006.trc20Token/index.ts]: Correct the id and name

File: packages/core-services/src/db/migrations/items/006.trc20Token/index.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  up: async db => {
+    await migrateTokenIdChangeInDb(db, idChanges);
+    await migrateTokenDetailsChangeInDb(db, changedCoins);
+    await resetContractTransactionBlockHeight(db, migration.name);
```
Comment (muzaffarbhat07) [inline packages/core-services/src/db/migrations/items/006.trc20Token/index.ts]: These functions work only for evm. Do update them for the tron as well. Or generalize them to work for any coin family.

File: packages/core-services/src/db/migrations/items/006.trc20Token/index.ts (hunk 1 above)
Comment (TejasvOnly) [reply packages/core-services/src/db/migrations/items/006.trc20Token/index.ts]: Reverting list update for tron.

File: packages/core-services/src/db/migrations/items/006.trc20Token/index.ts (hunk 2 above)
Comment (TejasvOnly) [reply packages/core-services/src/db/migrations/items/006.trc20Token/index.ts]: Reverting list update for tron.

**Review by muzaffarbhat07 — APPROVED** [review]

**Pull request merged**